.travis.yml

```diff
@@ -25,10 +25,11 @@ script:
         set -x
         # TRAVIS_BRANCH is name of the branch built OR the name of the target branch in case of PRs
         if [ "$TRAVIS_BRANCH" == "master" -a -z "$TRAVIS_PULL_REQUEST_BRANCH" ]; then
-          echo "Run sonar analysis for master only"
+          echo "Run jacoco analysis for master only"
           jacoco_goal="org.jacoco:jacoco-maven-plugin:prepare-agent"
-          sonar_goal="org.sonarsource.scanner.maven:sonar-maven-plugin:3.7.0.1746:sonar"
-          sonar_opts="-Dsonar.projectKey=openstack4j_openstack4j"
+          # disable sonar due to java 11 requirement
```

Comment:
Due to https://github.com/openstack4j/openstack4j/issues/150#issue-913170777 it's currently not possible to execute sonar analysis until java 11 is used. Therefore i disabled the sonar goal and sonar opts until this is fixed to still allow a running CI.